### PR TITLE
Update docs for adding new pipelines.

### PR DESCRIPTION
**Pull request by ewels**
Supersedes nf-core/nf-co.re#1682

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1837"></a>

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

File: markdown/developers/adding_pipelines.md [hunk 1]
```diff
@@ -48,50 +48,64 @@ These topics are specifically discussed in the `#new-pipelines` channel:
 ## Create a pipeline from the template
 
 You'll start by making a new pipeline locally and working with it on your own GitHub account.
-Only when it's ready do we move it to the nf-core GitHub organisation.
+Once you have a version of the pipeline that runs, ask the core team to move the repo to the nf-core GitHub organisation for final development, review and release.
```
Comment (mirpedrol) [inline markdown/developers/adding_pipelines.md:51]: Won't this encourage people to write for the first time in #new-pipelines with fully created pipelines? Which is then difficult to review for the 1st release

Comment (ewels) [reply markdown/developers/adding_pipelines.md:51]: I'll add a new section for this

Comment (ewels) [reply markdown/developers/adding_pipelines.md]: Added, please check 👍🏻

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
@nf-core-bot fix linting

File: markdown/developers/adding_pipelines.md (hunk 1 above)
Comment (mirpedrol) [reply markdown/developers/adding_pipelines.md]: looks good :)

**Pull request merged**